neuraxle/base.py

```diff
@@ -607,9 +636,10 @@ def __init__(
             self,
             steps_as_tuple: NamedTupleList,
             hyperparams: HyperparameterSamples = dict(),
-            hyperparams_space: HyperparameterSpace = dict()
+            hyperparams_space: HyperparameterSpace = dict(),
+            train: bool = True
     ):
-        super().__init__(hyperparams=hyperparams, hyperparams_space=hyperparams_space)
+        super().__init__(hyperparams=hyperparams, hyperparams_space=hyperparams_space, train=train)
```

Comment:
Same comment here: no need to set_train. 

If the user would really like to set train to false at the beginning (which would be quite rare), he could do this: 

- `my_step= TruncableStep([...]).set_train(is_train=False)`

This involves that `set_train` returns self.